api/views/perms/perms.py

```diff
@@ -38,11 +38,11 @@ def has_object_permission(self, request, view, obj):
 
 
 class CanSubmitRegistration(permissions.BasePermission):
-    message = _('To submit a registration, its state must be ACCEPTED.')
+    message = _('To submit a registration, its state must be ACCEPTED or DRAFT with no registration required.')
```

Comment:
C'est un message pour nous ou il est affiché à l'utilisateur? Si c'est le cas (affiché à l'utilisateur), on devrait peut être dire:
'To submit a registration, its state must be ACCEPTED. To submit an admission, its state must be DRAFT.'